modules/core/react/spec/InputProvider.spec.tsx

```diff
@@ -200,15 +200,18 @@ describe('InputProvider', () => {
     testInput(mockEvent, expectedInputType, expectedInputType, shimmedWindowProps);
   });
 
-  test(`nested input provider should not attach events and remove itself from the DOM`, () => {
+  test(`Multiple input provider should only attach once to the dom`, () => {
+    const ref = React.createRef<HTMLButtonElement>();
     const component = mount(
-      <InputProvider>
-        <InputProvider>
-          <h1>Test</h1>
-        </InputProvider>
-      </InputProvider>
+      <div>
+        <InputProvider></InputProvider>
```

Comment:
I'm surprised something didn't complain about not using `<InputProvider />` syntax here and below. No worries though.